### PR TITLE
[14.0][MIG] sale_global_discount

### PR DESCRIPTION
**Pull request by omar7r**
Depends on https://github.com/OCA/account-invoicing/pull/990

### CODE REVIEW

**Review by remi-filament — APPROVED** [review]
Functional test OK
Thanks for this migration !

**Comment by iwkse** [comment]
@remi-filament Hi, it's required to fix other's modules tests error to merge this PR?

**Comment by remi-filament** [comment]
Hi @iwkse this PR needs other PR to be merged before for modules needed by sale_global_discount :
- https://github.com/OCA/account-invoicing/pull/990 (account_global_discount)
- https://github.com/OCA/server-backend/pull/131 (base_global_discount)

**Comment by iwkse** [comment]
@omar7r @remi-filament 
Please can you rebase?
thank you

**Comment by omar7r** [comment]
@iwkse Rebased

File: sale_global_discount/README.rst [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+#. Go to *Settings > Technical > Parameters > Global Discounts*
```
Comment (francesco-ooops) [inline sale_global_discount/README.rst:41]: I would add for clarity

Go to Settings > Users, choose yours and set Manage Global Discounts.

(from base_global_discount configuration file)

**Review by francesco-ooops — APPROVED** [review]
LGTM

**Comment by iwkse** [comment]
@pedrobaeza is it ready for merge?

**Pull request merged**